playpen/vagrant-bootstrap-ansible.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+# minimal bootstrapping before kicking off ansible:
+#  install only what ansible needs to survive, or doesn't know how to do
+sudo dnf -y install python2 python2-dnf
```

Comment:
Can you:

1) rename this to bootstrap-ansible.sh?

and

2) Perhaps in a separate PR, but here is cool too if you prefer, have dev-setup.sh call this script too so that it will also keep working with F23?